src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -67,8 +80,9 @@ class AmendmentTableImpl : public AmendmentTable
 
     void addInitial () override;
 
-    AmendmentState* addKnown (const char* amendmentID, const char* friendlyName,
-        bool veto) override;
+    AmendmentState* addKnown (uint256 const& amendmentID,
```

Comment:
I'm guessing that no one derives from the `AmendmentTableImpl` and that is wasn't really designed to be further derived from (I could be wrong).  If I'm right, the `override` on these methods is good, but would be even better as `final`.  As an alternative you could just mark the class as `final`.  I depends on how easy you want to make it for someone to choose to derive from `AmendmentTableImpl`.